contentcuration/contentcuration/models.py

```diff
@@ -1982,6 +1982,9 @@ class Exercise(models.Model):
 class Invitation(models.Model):
     """ Invitation to edit channel """
     id = UUIDField(primary_key=True, default=uuid.uuid4)
+    accepted = models.BooleanField(default=False)
```

Comment:
At the risk of complicating this PR, would it make sense to just make this a status field? That way, it's easier to add new statuses and it implicitly enforces that invitations can only be accepted, pending, declined, or revoked?

Reply:
@jayoshih Kevin suggested the same, and I mulled that over for a long time but then decided against it, mainly because I didn't want to implement user-specific state evolution logic both on the front end and the back end, and in some sense, it'd be roughly equivalent to what's there now anyway.  At this point it'd be a pretty involved refactor to make that change at this stage.